cultcargo/casa/flag.yml

```diff
@@ -226,6 +226,7 @@ cabs:
                     - none
                     - apply
                     - calculate
+                default: apply
```

Comment:
Default is different from what the info string says. Do we need a default even? Preferred pattern is not to have default in CASA cabs so that it falls back on internal defaults.

Reply:
BTW, what does 'none' do? If 'none' is not a useful operation, alternative is to make this a ``required: true`` and no default.

Reply:
The default is 'apply' upon rereading. The documentation (https://casadocs.readthedocs.io/en/stable/api/tt/casatasks.flagging.flagdata.html#action) is ambiguous. It has 'none' as an option, but specifies '' as something that looks like none. I suggest we leave the default as 'apply' and let the user specify 'none' - but it's vague. For the moment have modified to have 'apply' as the default.

Reply:
I think just eliminate the default and push the ambiguity into CASA.

Reply:
Will do.